src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -166,6 +166,15 @@ def get_tables_to_migrate(self, tables_crawler: TablesCrawler) -> Collection[Tab
 
         return Threads.strict("checking all database properties", tasks)
 
+    def get_tables_to_acl(self, tables_crawler: TablesCrawler) -> Collection[TableToMigrate]:
```

Comment:
this method is not used anywhere. remove it.

Reply:
Removed

Reply:
it's not removed.